tests/unit/test_column_noise.py

```diff
@@ -132,11 +132,8 @@ def string_series():
     )
 
 
-def test_leave_blank(dummy_dataset):
-    config = get_configuration()[DATASETS.census.name][Keys.COLUMN_NOISE]["zipcode"][
-        NOISE_TYPES.leave_blank.name
-    ]
-    config.update(
+def test_leave_blank(dummy_dataset, fuzzy_checker):
```

Comment:
```suggestion
def test_leave_blank(dummy_dataset, fuzzy_checker: FuzzyChecker):
```

and do the associated import; this will get you tab-completion as Steve requested below.